tests/index.test.js

```diff
@@ -44,10 +49,35 @@ describe('serverless-lambda', () => {
       }
     }
 
-    await handler(event, context)
-    expect(context.done).toHaveBeenCalledWith(null, expect.objectContaining({
-      statusCode: 400
-    }))
+    const result = await handler(event, context)
+    expect(result.statusCode).toBe(400)
+    expect(spy).not.toHaveBeenCalled()
+  })
+
+  it('responds with a 400 when no x-github-event header is sent', async () => {
+    const event = {
+      body: {
+        installation: { id: 1 }
+      },
+      headers: {
+        'x-github-delivery': 123
+      }
+    }
+
+    const result = await handler(event, context)
+    expect(spy).not.toHaveBeenCalled()
+    expect(result.statusCode).toBe(400)
+  })
+
+  it('responds with a 400 when no headers are present', async () => {
```

Comment:
I like adding a reference to an issue/pull request so we don't have to wonder later why this exists

```suggestion
  it('responds with a 400 when no headers are present (#48)', async () => {
```

Reply:
Makes sense to me!